chalicelib_smaht/checks/wrangler_checks.py

```diff
@@ -1065,16 +1063,16 @@ def core_project_status(connection, **kwargs):
         check.action = 'share_core_project'
     else:
         check.status = 'PASS'
-        check.summary = 'All CGAP Core items are shared.'
-        check.description = ('All CGAP Core items are shared:'
+        check.summary = 'All smaht Core items are shared.'
```

Comment:
Not `SMaHT` throughout here? Lowercase looks funny next to upper-initial `Core`.